tests/storage/restore.go

```diff
@@ -745,8 +745,8 @@ var _ = SIGDescribe("[Serial]VirtualMachineRestore Tests", func() {
 				}, 360, 10).Should(Succeed())
 
 				b := append([]expect.Batcher{
-					&expect.BSnd{S: "cat /var/run/resize.rootfs | grep resized\n"},
-					&expect.BExp{R: "resized successfully"},
+					&expect.BSnd{S: "cat /var/run/resize.rootfs | grep -q resized && echo 'pass'\n"},
+					&expect.BExp{R: console.RetValue("pass")},
```

Comment:
Despite all the attempts to explain me the original  issue on the last PR where this go fixed, I still don't understand why only this test is affected. :D

 Does it have to do with a cirros start from a snapshot vs from a unmodified base image?

Reply:
The difference is that this test runs with a block storage vs filesystem, and with the block storage there is some resize happening in the bot of the vm(its before doing anything in the test before snapshot and the rest)

Reply:
Does it happen with the alpine image too? Maybe that is an easy fix to just switch the image?